src/bun.js/javascript.zig

```diff
@@ -2716,15 +2734,24 @@ pub const BuildError = struct {
         return ZigString.init(this.msg.data.text).toValue(ctx.ptr()).asRef();
```

Comment:
we should mlake all of these `toValueGC`

it's probably the cause of the crash you're seeing